Brewfile

```diff
@@ -1,5 +1,9 @@
+tap "taiki-e/tap"
```

Comment:
This might also be unnecessary in that case

Reply:
Removed in [0cb0dc8](https://github.com/NomicFoundation/hardhat/pull/4380/commits/0cb0dc81d17d19c1f20b00fc0918d2451b727e20)